src/databricks/labs/ucx/runtime.py

```diff
@@ -138,6 +139,20 @@ def assess_pipelines(cfg: WorkspaceConfig):
     crawler.snapshot()
 
 
+@task("assessment", depends_on=[setup_schema])
+def assess_aws_instance_profiles(cfg: WorkspaceConfig):
```

Comment:
```suggestion
def crawl_aws_instance_profiles(cfg: WorkspaceConfig):
```
There's no assessment done there